health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppHelpTutorial.js

```
@@ -0,0 +1,233 @@
+    <div
+      key={index}
+      className="tutorial-card"
+      onClick={() => window.open(helpContent?.url, "_blank")}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter' || e.key === ' ') {
+          e.preventDefault();
+          window.open(helpContent?.url, "_blank");
+        }
+      }}
+      tabIndex={0}
+      role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)